custom-hooks/Exercise1/solution/src/index.css

```diff
@@ -0,0 +1,69 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
```

Comment:
I haven’t run your code so I don’t know this is the case, but I’m guessing that in dark mode you’ll have a flash of white and then it’ll switch to dark when first loading the page. If so, the only way I know to fix that is to include the `color-scheme` line in your HTML: https://github.com/bitovi/trainings/blob/024e679bafa2a7275b253997b3036ad0412e7185/incorporating-accessibility-into-your-dev-process/forms/solution/index.html#L8-L14